postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java

```diff
@@ -287,8 +287,7 @@ private void handlePartialUpdate(Map<String, String> fileContents) {
                 if (Files.exists(existingFilePath)) {
                     try {
                         String existingFileContent = new String(Files.readAllBytes(existingFilePath), StandardCharsets.UTF_8);
-                        String updatedContent = PartialUpdateHandler.handlePartialUpdateForFile(generatedFileContent, existingFileContent);
-                        return updatedContent;
+                        return PartialUpdateHandler.handlePartialUpdateForFile(generatedFileContent, existingFileContent);
```

Comment:
nit, should we in future refactor it to use a class similar to `JavaFile` (i.e. have filename and content) as input parameter? So that it be much simpler to decide whether the file is package-info.java.